tardis/montecarlo/montecarlo_numba/calculate_distances.py

```diff
@@ -91,9 +92,8 @@ def calculate_distance_line(
     nu_diff = comov_nu - nu_line
 
     # for numerical reasons, if line is too close, we set the distance to 0.
-    if r_packet.is_close_line:
+    if abs(nu_diff/nu) < CLOSE_LINE_THRESHOLD:
```

Comment:
Looks like you left in your close line work. Should either be removed from this PR, or the PR should be set up to branch from that fix.